README.md

```diff
@@ -1,2 +1,43 @@
-Bring Your Own Notebook (BYON) project repository.
---------------------------------------------------
+# Bring Your Own Notebook (BYON) Working Group
+
+The goal of the Bring Your Own Notebook (BYON) working group is to create a service that implements the BYON MVP in the context of Open Data Hub (ODH).
+
+## Phase 1
+
+The Phase 1 (MVP) of BYON aims to implement the following requirements:
+
+### Requirements
+
+Functionality for an Admin to import existing images using a URL linked to a public registry through a UI and spawn a Jupyter Notebook image which is available for selection for users through the Spawner like out-of-the-box images.
+
+* [Ability to import notebook images using a URL via a UI](https://github.com/open-services-group/byon/issues/1)
+* [Validation on imported images to ensure they can be used](https://github.com/open-services-group/byon/issues/1#)
+* [Ability for Admins to Rename, Delete, Disable images](https://github.com/open-services-group/byon/issues/16)
+* [Imported Notebook images available for selection in Spawner UI](https://github.com/open-services-group/byon/issues/1)
+
+### Target
+
+We plan to add this Phase 1 service to Open Data Hub (ODH).
+* https://github.com/open-services-group/byon/issues/19
+* https://issues.redhat.com/browse/ODH-591
+
+
+## Next Phases
+
+* [Create new notebook image from scratch to be spawned from JH UI](https://github.com/open-services-group/byon/issues/2)
```

Comment:
Here I would say a similar thing, linking a milestone, when it will be decided in the next calls, wdyt? Because I think adding them here each time might be unmaintainable if we have 200 issues in the future.